service/storage_proxy.cc

```diff
@@ -7279,7 +7279,7 @@ future<> storage_proxy::cancel_write_handlers(noncopyable_function<bool(const ab
             // iterator_guard handles safe iterator updates while allowing prompt
             // handler destruction and client response.
             cancellable_write_handlers_list::iterator_guard ig{*_cancellable_write_handlers_list, it};
-            co_await maybe_yield();
+            co_await coroutine::maybe_yield();
```

Comment:
Sure, but we can just as well go over all `co_await maybe_yield();` and `co_await seastar::maybe_yield();` calls in the whole repo (I see 13) and change them. Better not to do it in a backported PR, though, so I'll leave it as an easy optional follow-up.